### PR TITLE
ENT-322 Creating an enterprise_worker service user.

### PR DESCRIPTION
**Pull request by asadiqbal08**
TASK: [ENT-322](https://openedx.atlassian.net/browse/ENT-322)

@saleem-latif  share your initial thoughts on that.

### CODE REVIEW

**Comment by asadiqbal08** [comment]
@clintonb, I am wondering, How can we then add this user in vagrant devstacks ? 
I have added a management command in docker devstack.

File: provision-lms.sh [hunk 1]
```diff
@@ -21,6 +21,10 @@ docker-compose exec lms bash -c 'source /edx/app/edxapp/edxapp_env && cd /edx/ap
 docker-compose exec lms bash -c 'source /edx/app/edxapp/edxapp_env && python /edx/app/edxapp/edx-platform/manage.py lms --settings=devstack_docker manage_user edx edx@example.com --superuser --staff'
 docker-compose exec lms bash -c 'source /edx/app/edxapp/edxapp_env && echo "from django.contrib.auth import get_user_model; User = get_user_model(); user = User.objects.get(username=\"edx\"); user.set_password(\"edx\"); user.save()" | python /edx/app/edxapp/edx-platform/manage.py lms shell  --settings=devstack_docker'
 
+# Create an enterprise service user for edxapp
+docker-compose exec lms bash -c 'source /edx/app/edxapp/edxapp_env && python /edx/app/edxapp/edx-platform/manage.py lms --settings=devstack_docker manage_user enterprise_worker enterprise_worker@example.com'
+docker-compose exec lms bash -c 'source /edx/app/edxapp/edxapp_env && echo "from django.contrib.auth import get_user_model; User = get_user_model(); user = User.objects.get(username=\"enterprise_worker\"); user.set_password(\"edx\"); user.save()" | python /edx/app/edxapp/edx-platform/manage.py lms shell  --settings=devstack_docker'
```
Comment (saleem-latif) [inline provision-lms.sh]: Why do we need to assign a password to the worker user ?

Comment (asadiqbal08) [reply provision-lms.sh]: Actually, I can see other worker users also have a password. Is it a bad idea?

**Review by saleem-latif — APPROVED** [review]
Overall LGTM 👍

**Comment by asadiqbal08** [comment]
@douglashall fyi

**Review by clintonb — APPROVED** [review]
Does this user require any special permissions?

**Comment by clintonb** [comment]
@asadiqbal08 if you want to get this on Vagrant devstacks, you'll need to add a similar call in edx/configuration.

**Pull request merged**